command/agent/http.go

```diff
@@ -783,10 +783,17 @@ func parseWait(resp http.ResponseWriter, req *http.Request, b *structs.QueryOpti
 }
 
 // parseConsistency is used to parse the ?stale query params.
-func parseConsistency(req *http.Request, b *structs.QueryOptions) {
+func parseConsistency(resp http.ResponseWriter, req *http.Request, b *structs.QueryOptions) {
 	query := req.URL.Query()
-	if _, ok := query["stale"]; ok {
-		b.AllowStale = true
+	if staleVal, ok := query["stale"]; ok {
+		if staleVal[0] == "true" || staleVal[0] == "" {
+			b.AllowStale = true
+		} else if staleVal[0] == "false" {
+			// fall through
```

Comment:
What if we re-organize these cases as a switch statement? Comments about control flow are usually a good sign there's a better pattern to be found